Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs

```diff
@@ -252,18 +252,26 @@ private void RotateToNextItem()
                 sb.Children.Add(anim);
             }
 
-            sb.Completed += (a, b) =>
+            sb.Completed += async (a, b) =>
             {
+                if (_currentElement != null)
+                {
+                    _currentElement.DataContext = _nextElement.DataContext;
+                }
+
+                // make sure DataContext on _currentElement has had a chance to update the binding
+                // avoids flicker on rotation
+                await System.Threading.Tasks.Task.Delay(50);
```

Comment:
Would it be better to use the dispatcher here to queue up in the UI thread for processing?

Reply:
Tried that, it didn't have an effect at all :(

Reply:
Normal or Low priority?

If we need to keep this delay, we should subtract it from the rotation time, eh?

Reply:
Good point, only tried normal, let me try low. On the subtraction, don't think it's needed as it's random anyway

Reply:
Bummer, that's frustrating.  I know Idle usually causes bad things in UWP land.

For the timing though, it's only random is the [ExtraRandomDuration](https://github.com/Microsoft/UWPCommunityToolkit/blob/f9f5c6a5a9bceeccc8d381a836231ee429527ee7/Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs#L411) is non-zero.  Otherwise, it's fixed, so we would want to subtract the Delay time (as a const) from the base RotationDelay.  

We should probably clarify in the doc comments for the property too that there's a minimum delay.

Reply:
Go for it and we should be good!

Reply:
The Task.Delay doesn't really add any timing to the duration, it happens in parallel. When the rotation completes (after the duration) it just waits 50ms before switching the two items. The duration is not affect by this at all. 

I did add a quick snippet about the duration in the documentation though.